habitat/tasks/rearrange/actions.py

```diff
@@ -126,6 +129,40 @@ def step(self, vel, should_step=True, *args, **kwargs):
             return None
 
 
+@registry.register_task_action
+class ArmAbsPosAction(SimulatorTaskAction):
+    @property
+    def action_space(self):
+        return spaces.Box(shape=(7,), low=0, high=1, dtype=np.float32)
```

Comment:
Magical number 7 should have reasonable constant name so we can change it easily when needed and distinguish it from other 7s.